docs/img/logomark_purser_dark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 85.49 113.64"><title>logo_</title><g id="logos"><path d="M42.74,32.88C19.15,32.88,0,52,0,72.65v41H85.49v-38A42.75,42.75,0,0,0,42.74,32.88Zm0,56.6A16.21,16.21,0,1,1,59,73.27,16.22,16.22,0,0,1,42.74,89.48Z" style="fill:#2e4153"/><polygon points="71 0 42.75 28.25 14.49 0 71 0" style="fill:#2e4153"/></g></svg>
```

Comment:
This SVG
- was not flatten
- has decimal point viewbox sizing
- doesn't have a eof line

Please use this one:
```svg
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 86 114">
   <path style="fill:#2e4153" d="M 14.576172,0 43.005859,28.337891 71.423828,0 Z M 42.994141,32.980469 C 19.263412,32.980469 0,52.159453 0,72.873047 V 114 H 86 V 75.882812 A 43.005029,42.881653 0 0 0 42.994141,32.980469 Z M 42.417969,57.248047 A 16.306702,16.25992 0 0 1 59.351562,73.496094 16.316762,16.269951 0 0 1 42.994141,89.755859 16.306702,16.25992 0 0 1 42.417969,57.248047 Z" />
</svg>
```